examples/mega-form/src/App.tsx

```diff
@@ -74,8 +89,8 @@ const Form = ({
       </div>
 
       <ul>
-        {fieldAtoms.map(([fieldAtom, remove]) => (
-          <li>
+        {fieldAtoms.map(([fieldAtom, remove], index) => (
+          <li key={index}>
```

Comment:
```suggestion
        {fieldAtoms.map(([fieldAtom, remove]) => (
          <li key={`${fieldAtom}`}>
```
I hope this works. Does it?